sys-libs/mesa/mesa-17.1.10.recipe

```diff
@@ -104,6 +106,7 @@ BUILD()
 		--disable-gles2 \
 		--disable-shared-glapi \
 		--with-gallium-drivers=swrast \
+		--enable-texture-float \
```

Comment:
This is to document in the commit message.

Reply:
@kenmays you have still not documented why this was done.

Reply:
Allows use of floating point texture data.